examples/longan_nano_led.asm

```diff
@@ -38,8 +38,7 @@ gpio_init:
 
 gpio_init_config:
     # multiply pin number by 4 to get shift amount
-    addi t1, zero, 4
-    mul a1, a1, t1
+    slli a1,a1,2
```

Comment:
This is a great change! The only thing I'd add are the spaces between the instruction args (just for consistency).

Reply:
Thank you. I corrected the formatting accordingly.